src/handlers/organizations.py

```
@@ -30,266 +34,275 @@ async def handle_organizations(
+        count_query = f'''
+            SELECT COUNT(*) as total FROM organization o WHERE {where_sql}
+        '''
+        count_result = await db.prepare(count_query).bind(*bind_params[:-2]).first()
+        count_data = count_result.to_py() if hasattr(count_result, 'to_py') else dict(count_result) if count_result else {}
+        total = count_data.get("total", 0)
     
-    return paginated_response(organizations, page=page, per_page=per_page, total=total)
+        return paginated_response(organizations, page=page, per_page=per_page, total=total)
+
+    except (OSError, TimeoutError) as e:
+        logger.error(f"Database error in organizations handler: {str(e)}")
```

Ruff (0.15.5)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)